drivers/wifi/winc1500/wifi_winc1500.c

```diff
@@ -305,7 +305,11 @@ static int winc1500_get(sa_family_t family,
 		return -1;
```

Comment:
In the commit subject, could you write `drivers: wifi: winc1500: .....` instead, as there is no need to mention wifi in the driver name.